rm-datasource/src/main/java/io/seata/rm/datasource/exec/ExecuteTemplate.java

```diff
@@ -103,10 +103,10 @@
 
         } catch (Throwable ex) {
             if (ex instanceof SQLException) {
-                throw (SQLException)ex;
+                throw (SQLException) ex;
```

Comment:
how about transfer the ex (non SQLException) to sqlException in the if{} , then remove the else and throw the ex directly ?

Reply:
ok，done